server/src/test/java/org/opensearch/index/compositeindex/datacube/startree/StarTreeTestUtils.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.compositeindex.datacube.startree;
+
+import org.opensearch.index.compositeindex.datacube.startree.aggregators.numerictype.StarTreeNumericType;
+
+public class StarTreeTestUtils {
```

Comment:
This class is going to hold many methods for verification of file formats and doc values in the subsequent PRs. Please refer to https://github.com/opensearch-project/OpenSearch/pull/14809 for broader context. Thank you!